src/modules/organisation-role/organisation-role.controller.ts

```diff
@@ -89,6 +94,18 @@ export class OrganisationRoleController {
     }
   }
 
+  @Delete(':id/roles:roleId')
+  @UseGuards(AuthGuard, OwnershipGuard)
+  @HttpCode(HttpStatus.OK)
+  @ApiOperation({ summary: 'Delete a role in an organisation' })
+  @ApiParam({ name: 'organisationId', required: true, description: 'ID of the organisation' })
+  @ApiResponse({ status: 200, description: 'Role successfully removed' })
+  @ApiResponse({ status: 400, description: 'Invalid role ID format' })
+  @ApiResponse({ status: 404, description: 'Role not found' })
+  async remove(@Param('id') organisationId: string, @Param('roleId') roleId: string) {
```

Comment:
Use a more descriptive controller name

```suggestion
  async removeRole(@Param('id') organisationId: string, @Param('roleId') roleId: string) {
```